ui/Screen/Project/Source/CheckoutButton.svelte

```diff
@@ -30,9 +30,9 @@
     margin-top: 3rem;
     right: 0;
     position: absolute;
-    border-radius: 0.5rem;
+    border-radius: 1rem;
     background: var(--color-background);
-    border: 1px solid var(--color-foreground-level-3);
+    /* border: 1px solid var(--color-foreground-level-3); */
```

Comment:
If this is intentional, we should just delete the line instead of commenting it out.

Reply:
Ah not intentional. Will do